Fixing problems when imposing Bofang + Fixed Temperature

joaquinirazabal opened the pull request:
When freeing the nodes the imposed temperature condition was removed, this change corrects the problem

joaquinirazabal committed:
Deleting deprecated variables

joaquinirazabal committed:
Adding new process to impose initial temperature

joaquinirazabal committed:
Merge branch 'master' into feature-dam-impose-temperature

joaquinirazabal committed:
Deleting deprecated variables

joaquinirazabal committed:
Aesthetics

joaquinirazabal committed:
Correcting problem with initial Temperature

joaquinirazabal committed:
Merge remote-tracking branch 'origin/feature-dam-impose-temperature' into feature-dam-impose-temperature

joaquinirazabal committed:
Merge branch 'master' into feature-dam-impose-temperature

joaquinirazabal committed:
Fixing bug

joaquinirazabal committed:
Merge branch 'master' into feature-dam-impose-temperature

ipouplana commented on applications/DamApplication/python_scripts/impose_face_heat_flux_process.py:
```diff
@@ -18,18 +18,16 @@ def __init__(self, Model, settings ):
             else:
-                t_uniform.AddValue("table",settings["table"])
-                self.components_process_list.append(ApplyDoubleTableProcess(model_part, t_uniform))
+                self.components_process_list.append(DamFixTemperatureConditionProcess(model_part, settings))
```
Why do you define DamFixTemperatureConditionProcess in impose_face_heat_flux_process.py ? Probably I'm forgetting something...

joaquinirazabal replied:
Hi @ipouplana! As far as I am concerned, this is necessary because although using heat flux you need to specify the ambient temperature.

ipouplana replied:
Ok, thanks!

ipouplana reviewed (commented):
Hi @joaquinirazabal , if I understood correctly, in the new process (DamFixTemperatureConditionProcess) you fix the temperature at the Dirichlet nodes every time step, so that there are no problems when freeing them in the Bofang process, right?

joaquinirazabal committed:
Correcting error when the temperature is constant and fixed

Correcting error when the temperature is constant and fixed over all the calculation

joaquinirazabal commented:
Thank you for your review.

Merged!
